torch/distributed/rpc/__init__.py

```diff
@@ -198,6 +198,25 @@ def init_rpc(
         _init_rpc_backend(backend, store, name, rank, world_size, rpc_backend_options)
 
 
+    @api._require_initialized
+    def barrier(worker_names=None):
+        r"""
+        Synchronizes local and remote RPC processes.
+
+        This will block until all local and remote RPC processes specified under worker_names
+        reach this method to wait for all outstanding work to complete.
+
+        Args:
+            worker_names (List[str], optional): The set of workers to synchronize. If ``None``, the
+            set will be all workers. Default is ``None``.
```

Comment:
we might need one more indent on the second line. Shall we build the docs to verify this can render correctly?